src/featureSelectors.test.ts

```diff
@@ -29,13 +29,40 @@ describe("featureSelectors", () => {
         name: "users",
         entity: "users",
       });
+
       const output = getDenormalizedData.resultFunc(featureSlices, true);
+
       expect(output).toEqual({
         users: [
           { id: 1, name: "Garrett" },
           { id: 2, name: "Laura" },
         ],
       });
     });
+
+    it("should return an empty object", () => {
```

Comment:
the test name here should be more description to say what you're trying to test. why should it return an empty object? What condition is this testing?